Assignment2/Josesilva99-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,58 @@
+{
+
+"@context": {
+		"base": "http://example.org/assigment2/",
+		"Class01": "base:Class01",
+		"Sensor029": "Class01:Sensor029",
+		"Computer101": "Class01:Computer101",
+		"Measurement8401": "Sensor029:Measurement8401",
+		"User10A": "Computer101:User10A",
+		"includes": "Class01:Includes",
+		"hasMeasurement": "Sensor029:hasMeasurement",
+		"hasTemperature": "Measurement8401:hasTemperature",
+		
+		"hasOwner": "Computer101:hasOwner",
+		"hasName": "User10A:hasName",
+		"atTime":"Measurement8401:atTime",

```

Comment:
These attribute definitions work, but semantically they aren't ok. Append a definition with others might cause errors e.g. "Class01:Sensor029". The best way is to include the prefix and then append the name of the concept or attribute e.g. "Sensor029":base:Sensor029".
You don't need to change them, but remember to provide good definitions for the future.